support-frontend/app/lib/CustomHttpErrorHandler.scala

```diff
@@ -36,7 +36,8 @@ class CustomHttpErrorHandler(
         "Error 404",
         EmptyDiv("error-404-page"),
         Left(RefPath("error404Page.js")),
-        Left(RefPath("error404Page.css"))
+        Left(RefPath("error404Page.css")),
+        None
```

Comment:
Preferably, we wouldn't pass in `None` here, and then `fontLoaderBundle` wouldn't have to be an option.